tests/models/test_model.py

```diff
@@ -1901,7 +1901,7 @@ def test_hf_init(tmp_path: pathlib.Path,
     precision = Precision('amp_bf16')
 
     hf_config = MPTConfig(
-        init_device=init_device,
+        init_device='cpu',
```

Comment:
We are still testing all the `init_device`s in the parametrization, just later down in the test. This first model creation is just so that we can call `save_pretrained` to get a model to call `from_pretrained` on.